index.html

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Slide Show</title>
+		<style>
+		body{
+
+		}
+		.slideShow-container{
+			background-color: rgba(255,255,255,0.3);
+			position: absolute;
+			height: 95%;
+			width: 100%;
+			top: 10%;
+			left: 0%;
+		}
+		.slides-container{
+			position: absolute;
+			height: 100%;
+    		width: 100%;
+    		top: 0;
+    		left: 0;
+
+		}
+		.btn{
+			background-repeat: no-repeat;
+    		position: absolute;
+    		top: 0;
+    		height: 100%;
+    		width: 4%;
+    		cursor: pointer;
+    		opacity: 0.3;
+		}
+		.Option-bar{
+			background-color: rgba(0,0,0,0.5);
+			position: absolute;
```

Comment:
You could group common classes that use the position absolute attribute

Reply:
**How? What do you mean?**